PagePixels Screenshots (Certified Publisher)

Anthony-Dev opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

Anthony-Dev committed:
PagePixels Screenshots (Certified Publisher)

* - Initial Commit

Anthony-Dev committed:
Update Readme to match the template's structure, remove the unnecessary logo.png file.

Anthony-Dev committed:
Update the readme to include the Deployment Instructions. Update the apiDefinition to fix typos.

Anthony-Dev committed:
Add the x-ms-connector-metadata property to the ApiDefinition.swagger.json to define the Website, Privacy Policy, and Categories.

vmanoharas commented:
Hello @Anthony-Dev, 

Thank you for creating new PR.  However, there are multiple swagger validation errors.  Kindly fix them. 

2025-09-01T14:44:10.4625447Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/2
2025-09-01T14:44:10.4666845Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/3
2025-09-01T14:44:10.4668695Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/4
2025-09-01T14:44:10.4669948Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/5
2025-09-01T14:44:10.4671085Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/6
2025-09-01T14:44:10.4672039Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/7
2025-09-01T14:44:10.4673001Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/8
2025-09-01T14:44:10.4673938Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/9
2025-09-01T14:44:10.4674903Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/10
2025-09-01T14:44:10.4675885Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/11
2025-09-01T14:44:10.4680323Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/12
2025-09-01T14:44:10.4683025Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/13
2025-09-01T14:44:10.4684675Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/14
2025-09-01T14:44:10.4688760Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/15
2025-09-01T14:44:10.4691717Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/16
2025-09-01T14:44:10.4693886Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/17
2025-09-01T14:44:10.4695329Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/18
2025-09-01T14:44:10.4697680Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/19
2025-09-01T14:44:10.4699901Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/20
2025-09-01T14:44:10.4703365Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/21
2025-09-01T14:44:10.4706354Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/22
2025-09-01T14:44:10.4708570Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/23
2025-09-01T14:44:10.4710777Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/24
2025-09-01T14:44:10.4712729Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/25
2025-09-01T14:44:10.4714959Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/26
2025-09-01T14:44:10.4718363Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/27
2025-09-01T14:44:10.4720645Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/28
2025-09-01T14:44:10.4722635Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/29
2025-09-01T14:44:10.4725516Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/30
2025-09-01T14:44:10.4728451Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/31
2025-09-01T14:44:10.4730605Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/32
2025-09-01T14:44:10.4732972Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/33
2025-09-01T14:44:10.4734556Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/34
2025-09-01T14:44:10.4735814Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap/get/parameters/35
2025-09-01T14:44:10.4737154Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/2
2025-09-01T14:44:10.4738361Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/3
2025-09-01T14:44:10.4739553Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/4
2025-09-01T14:44:10.4740732Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/5
2025-09-01T14:44:10.4741917Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/6
2025-09-01T14:44:10.4743078Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/7
2025-09-01T14:44:10.4744283Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/8
2025-09-01T14:44:10.4745481Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/9
2025-09-01T14:44:10.4747902Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/10
2025-09-01T14:44:10.4749400Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/11
2025-09-01T14:44:10.4750638Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/12
2025-09-01T14:44:10.4751900Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/13
2025-09-01T14:44:10.4753003Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/14
2025-09-01T14:44:10.4754098Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/15
2025-09-01T14:44:10.4755206Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/16
2025-09-01T14:44:10.4756644Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/17
2025-09-01T14:44:10.4757776Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/18
2025-09-01T14:44:10.4758883Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/19
2025-09-01T14:44:10.4760271Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/20
2025-09-01T14:44:10.4761379Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/21
2025-09-01T14:44:10.4762466Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/22
2025-09-01T14:44:10.4763564Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/23
2025-09-01T14:44:10.4764690Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/24
2025-09-01T14:44:10.4765642Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/25
2025-09-01T14:44:10.4766880Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/26
2025-09-01T14:44:10.4767961Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/27
2025-09-01T14:44:10.4769073Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/28
2025-09-01T14:44:10.4770042Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/29
2025-09-01T14:44:10.4771000Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/30
2025-09-01T14:44:10.4772022Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/31
2025-09-01T14:44:10.4772985Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/32
2025-09-01T14:44:10.4773959Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/33
2025-09-01T14:44:10.4774918Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/34
2025-09-01T14:44:10.4775892Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_html/get/parameters/35
2025-09-01T14:44:10.4777107Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/2
2025-09-01T14:44:10.4778140Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/3
2025-09-01T14:44:10.4779149Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/4
2025-09-01T14:44:10.4780129Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/5
2025-09-01T14:44:10.4781131Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/6
2025-09-01T14:44:10.4782125Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/7
2025-09-01T14:44:10.4783172Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/8
2025-09-01T14:44:10.4784435Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/9
2025-09-01T14:44:10.4787286Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/10
2025-09-01T14:44:10.4788992Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/11
2025-09-01T14:44:10.4790292Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/12
2025-09-01T14:44:10.4791462Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/13
2025-09-01T14:44:10.4792633Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/14
2025-09-01T14:44:10.4793798Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/15
2025-09-01T14:44:10.4794947Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/16
2025-09-01T14:44:10.4796574Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/17
2025-09-01T14:44:10.4797936Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/18
2025-09-01T14:44:10.4799084Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/19
2025-09-01T14:44:10.4800238Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/20
2025-09-01T14:44:10.4802230Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/21
2025-09-01T14:44:10.4803685Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/22
2025-09-01T14:44:10.4805072Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/23
2025-09-01T14:44:10.4806691Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/24
2025-09-01T14:44:10.4807903Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/25
2025-09-01T14:44:10.4809056Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/26
2025-09-01T14:44:10.4810213Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/27
2025-09-01T14:44:10.4811353Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/28
2025-09-01T14:44:10.4812516Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/29
2025-09-01T14:44:10.4813669Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/30
2025-09-01T14:44:10.4814928Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/31
2025-09-01T14:44:10.4816010Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/32
2025-09-01T14:44:10.4818840Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/33
2025-09-01T14:44:10.4820043Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/34
2025-09-01T14:44:10.4821287Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/35
2025-09-01T14:44:10.4822367Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_real_location/get/parameters/36
2025-09-01T14:44:10.4823451Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/3
2025-09-01T14:44:10.4824533Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/4
2025-09-01T14:44:10.4825583Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/5
2025-09-01T14:44:10.4826900Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/6
2025-09-01T14:44:10.4828088Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/7
2025-09-01T14:44:10.4829204Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/8
2025-09-01T14:44:10.4830173Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/9
2025-09-01T14:44:10.4831159Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/10
2025-09-01T14:44:10.4832143Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/11
2025-09-01T14:44:10.4833106Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/12
2025-09-01T14:44:10.4834434Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/13
2025-09-01T14:44:10.4835756Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/14
2025-09-01T14:44:10.4837011Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/15
2025-09-01T14:44:10.4838147Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/16
2025-09-01T14:44:10.4839252Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/17
2025-09-01T14:44:10.4840374Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/18
2025-09-01T14:44:10.4841724Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/19
2025-09-01T14:44:10.4844971Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/20
2025-09-01T14:44:10.4846551Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/21
2025-09-01T14:44:10.4847587Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/22
2025-09-01T14:44:10.4848591Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/23
2025-09-01T14:44:10.4849813Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/24
2025-09-01T14:44:10.4850940Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/25
2025-09-01T14:44:10.4852236Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/26
2025-09-01T14:44:10.4853436Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/27
2025-09-01T14:44:10.4854647Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/28
2025-09-01T14:44:10.4855847Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/29
2025-09-01T14:44:10.4857189Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/30
2025-09-01T14:44:10.4858399Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/31
2025-09-01T14:44:10.4859605Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/32
2025-09-01T14:44:10.4860805Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/33
2025-09-01T14:44:10.4862005Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/34
2025-09-01T14:44:10.4863210Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/35
2025-09-01T14:44:10.4864506Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/36
2025-09-01T14:44:10.4865495Z ##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1snap_ai_analysis/get/parameters/37
2025-09-01T14:44:10.4867090Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1snap/get/responses/default
2025-09-01T14:44:10.4868737Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1snap_html/get/responses/default
2025-09-01T14:44:10.4870097Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1snap_real_location/get/responses/default
2025-09-01T14:44:10.4871435Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1snap_ai_analysis/get/responses/default
2025-09-01T14:44:10.4873024Z ##[error]The 'default' response should not have schema definition. Schemas should be defined on expected responses only. ValuePath: /paths/~1real_locations/get/responses/default
2025-09-01T14:44:10.4874148Z ##[error]The 'contact' property is required. ValuePath: /info

Anthony-Dev committed:
Merge branch 'dev' of https://github.com/microsoft/PowerPlatformConnectors into dev

Anthony-Dev committed:
Update PagePixels Screenshots Swagger for Power Platform

- Add contact info (name, URL, email) to OpenAPI info
- Replace generic “default” with explicit 200 responses and “Screenshot created”
- Add x-ms-summary across parameters and response fields for better Power Automate UX
- Clarify field descriptions (IDs, links, timestamps) and AI analysis object
- Standardize parameter summaries (format/quality/scale, geolocation, headers/cookies, TTL, blockers)
- Add “Real location” summary and improve geolocation endpoint parameter clarity
- Add default “Unexpected error” for locations listing endpoint
- No functional API changes; documentation/metadata improvements only

Anthony-Dev commented:
Hey @vmanoharas  - No problem, I believe the latest commit has resolved the swagger validation errors.

Anthony-Dev commented:
Hey @vmanoharas - Just checking in - everything should be good to go now - please let me know if there is anything else I can help with on my end.

vmanoharas reviewed (approved):
Dear Partner,
 
Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. 
 
As next steps:
 
Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
Validate the package for structure before submitting the package for certification to Partner Center - [Run Package Validator tool](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission%23step-7-validate-the-package-for-structure&data=05%7C02%7Cshefaali.patankar%40microsoft.com%7C6f0c4adf688f42a7107908dcb6c72a8c%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638586213737574171%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=9QGRIRe%2FZ%2B1sgUF5JQjY2DeuyxDt8f8lDwpGFEaaBmg%3D&reserved=0)
Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0) and also [1000 Marketplace policies](https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#100-general)(as applicable)
Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)
 
For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)
 
Many thanks,
Power Platform
Copilot connector & plugin certification team
